app/models.py

```diff
@@ -0,0 +1,106 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from sqlalchemy.ext.declarative import declared_attr
+import sqlalchemy as sa
+
+migrate = Migrate()
+
+
+class IdModel(object):
```

Comment:
Didn't know you could do this! Just read the docs though and v cool :)

Reply:
Isn't it! Saves on some boilerplating